content/en/build/building-with-lotus-APIs.md

```diff
@@ -0,0 +1,646 @@
+   - query basic blockchain info
+   - manage Lotus wallet
+   - manage light wallet
+   - make a storage deal
+
+## Prerequisites
+
+Before getting started, you should be familiar with:
+
+- [How Filecoin works](https://docs.filecoin.io/about-filecoin/how-filecoin-works/)
+- [How to install a Lotus node for the Calibration network](https://lotu.sh/lotus/install/prerequisites/#supported-platforms)
+- [How to use Lotus command line](https://lotu.sh/lotus/manage/lotus-cli/)
+
+Also, make sure you have all the dependencies installed:
+
+- Fully Synced Calibration TestNet Lotus node
+- node.js
+- npm
+
+## Setup
+
+Before you start writing code to interact with hosted local Lotus node, you should make sure everything is set up properly. Especially ensure you have a local Lotus node fully synced for Calibration TestNet. 
+
+### Set up lotus node
+
+If you have not installed a Lotus node locally and fully synced for Calibration TestNet, you should go back and do it following the guild to [Install and launch a Lotus node](https://lotus.filecoin.io/docs/set-up/install/).
+
+To sum up, you need to:
+
+1. Download Lotus source code for the latest release
+
+   ```shell
+   git clone https://github.com/filecoin-project/lotus.git
+   cd lotus/
+   git checkout <vX.X.X> # tag for a release, currently v1.15.0
+   ```
+
+2. Build and install Lotus for Calibnet
+
+   ```shell
+   make clean calibnet # Calibration with min 32GiB sectors
+   sudo make install
+   ```
+
+3. Fast sync using latest snapshot (still need several hours to fully sync the chain)
+
+   ```shell
+   lotus daemon --import-snapshot <path-to-calibration-snapshot>
+   ```
+
+4. [Create a wallet](https://lotu.sh/lotus/manage/manage-fil/#creating-a-wallet) and request some TestNet FIL from [Faucet](https://faucet.calibration.fildev.network/)
+
+4. [Enable remote API access to your Lotus node](https://lotu.sh/developers/api-access/)
+
+If everything set up correctly, you should be able to verify to:
+
+- make sure that everything is working correctly, check the status of your local node:
+
+  ```shell with-output
+  #Check if the node sync to the highest tipset
+  lotus sync wait
+  ```
+
+  ```shell
+  Worker: 10161; Base: 778820; Target: 778820 (diff: 0)
+  State: complete; Current Epoch: 778820; Todo: 0
+  
+  Done!
+  ```
+
+- Check your wallet balance
+
+  ```shell with-output
+  #Check wallet balance
+  lotus wallet list 
+  ```
+
+  ```shell
+  Address                                    Balance        Nonce  
+  t154xvuihhicgluafwmohwzwmtmqp44pwwgewyvma  100 FIL          0
+  ```
+
+### Create a node.js project
+
+Let's also set up a Node.js project with necessary dependencies, we will write basic javascript functions to focus more on learning the code to interact with Filecoin Network. 
+
+1. Create a new node.js project
+
+   ```shell
+   mkdir build-with-lotus && cd build-with-lotus
+   npm init #Initialize an project
```

Comment:
```suggestion
   npm init
```

Try not to comment your code. If something need explaining, explain it in a regular paragraph.